File: 2253/2253.cs
public class Solution {
    public int DynamicUnpivotingOfATable(TreeNode root) {
        if (root == null) return 0;
        return 1 + Math.Max(DynamicUnpivotingOfATable(root.left), DynamicUnpivotingOfATable(root.right));
    }
}

File: 2253/2253.cpp
class Solution {
public:
    int dynamicUnpivotingOfATable(TreeNode* root) {
        if(!root) return 0;
        return 1+max(dynamicUnpivotingOfATable(root->left),dynamicUnpivotingOfATable(root->right));
    }
};